.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: File metadata
         run: npm run check:metadata
       - name: Check images
-        run: npm run check:images
+        run: npm run check:images -- --apis
```

Comment:
Insted, this should go into https://github.com/Qiskit/documentation/blob/main/.github/workflows/api-checks.yml. We want the main job to be as fast as possible for non-API changes. We're okay with API changes being slightly slower (repeating the check of non-API)